woocommerce/woocommerce-gateway-stripe
# fix: payment methods checkbox alignment

# Changes proposed in this Pull Request:

I noticed that the vertical alignment of the checkboxes in the payment methods list is a bit off.

Helps with https://github.com/woocommerce/woocommerce-gateway-stripe/issues/1874 - splitting into multiple PRs

Before:
![Screen Shot 2021-10-04 at 5 18 58 PM](https://user-images.githubusercontent.com/273592/135932395-d472f2ba-934e-40d4-8afb-36eaae1e3f6a.png)


After:
![Screen Shot 2021-10-04 at 5 19 31 PM](https://user-images.githubusercontent.com/273592/135932444-6f182aac-03d7-49a7-ae44-9513f8cc080f.png)



# Testing instructions
- Ensure you have the `_wcstripe_feature_upe_settings` flag enabled
- Ensure you have UPE enabled
- Go to http://localhost:8082/wp-admin/admin.php?page=wc-settings&tab=checkout&section=stripe
- Alignment is improced
